modules/rdg/src/userobjects/ElementLoopUserObject.C

```diff
@@ -185,10 +185,31 @@ ElementLoopUserObject::onInternalSide(const Elem * elem, unsigned int side)
 }
 
 void
-ElementLoopUserObject::onInterface(const Elem * /*elem*/,
-                                   unsigned int /*side*/,
-                                   BoundaryID /*bnd_id*/)
+ElementLoopUserObject::onInterface(const Elem * elem, unsigned int side, BoundaryID /*bnd_id*/)
 {
+  _current_elem = elem;
+  // Pointer to the neighbor we are currently working on.
+  _current_neighbor = elem->neighbor(side);
+
+  // Get the global id of the element and the neighbor
+  const dof_id_type elem_id = elem->id();
+  const dof_id_type neighbor_id = _current_neighbor->id();
+
+  // TODO: add if-statement to check if this needs to be executed
+  if ((_current_neighbor->active() && (_current_neighbor->level() == elem->level()) &&
+       (elem_id < neighbor_id)) ||
+      (_current_neighbor->level() < elem->level()))
```

Comment:
Since the operation at the interface should closely resemble that of an internal side, this bit of code was taken from `onInternalSide()`.  We should get input from the original author (@yidongxiainl), but that is how I understand the logic.  I don't see in the documentation that `neighbor()` only returns an element at the same level.  Switching to use `neighbor_ptr()` seems a simple fix.